buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditTask.java

```diff
@@ -391,7 +391,7 @@ private String runForbiddenAPIsCli() throws IOException {
     private Configuration getRuntimeConfiguration() {
         Configuration runtime = getProject().getConfigurations().findByName("runtimeClasspath");
         if (runtime == null) {
-            return getProject().getConfigurations().getByName("testCompile");
+            return getProject().getConfigurations().getByName("testCompileClasspath");
```

Comment:
I assume this is to deal with standalone test projects, that don't have a "main" sourceset. If so, should we be using `testRuntimeClasspath` here? I dunno, @rjernst, what do you think? Should we even be auditing third party libraries for qa projects? I think we were discussion what all precommit checks should apply to those recently.

Reply:
I don't think we need third part audit for any tests, just our production code.

Reply:
Ok, let's leave this as-is for now, and we can rip this out when we refactor the standalone test plugin.